esp/templates/survey/answers.html

```diff
@@ -23,8 +23,16 @@
                         {% with "format=html&max="|concat:params.number_of_ratings as args %}
                             {{ q.answers|unpack_answers|histogram:args|safe }}
                         {% endwith %}
+                    {% elif q.question.question_type.name == "Yes-No Response" %}
+                        {% with "format=html&opts=Yes|No" as args %}
+                            {{ q.answers|unpack_answers|histogram:args|safe }}
+                        {% endwith %}
                     {% else %}
-                        {{ q.answers|unpack_answers|histogram:"format=html"|safe }}
+                        {% with params.list|join:"|" as opts %}
+                            {% with "format=html&opts="|concat:opts as args %}
+                                {{ q.answers|unpack_answers|histogram:args|safe }}
+                            {% endwith %}
+                        {% endwith %}
                     {% endif %}
```

Comment:
Please add a comment indicating which question type(s) the "else" block addresses, so that it will be clearer what needs to change if a new question type is added.